tests/locale/keymap_or_locale.pm

```diff
@@ -76,18 +71,30 @@ sub verify_default_keymap_x11 {
 }
 
 sub run {
-    # uncomment in case of different keyboard than us is used during installation ( feature not ready yet )
-    # my $expected   = get_var('INSTALL_KEYBOARD_LAYOUT','us');
-    my $expected       = 'us';
-    my %keystroke_list = (
-        us => '`1234567890-=~!@#$%^&*()_+',
-        fr => '²&é"(-è_çà)=~1234567890°+',
-        de => '1234567890ß°!"§$%&/()=?',
-        cz => ';+ěščřžýáíé=1234567890%'
-    );
-    my $keystrokes = $keystroke_list{$expected};
-
-    if (check_var('DESKTOP', 'textmode')) {
+    my $expected = get_var('INSTALL_KEYBOARD_LAYOUT', 'us');
+    # feature of switching keyboard during installation is not ready yet,
+    # so when typing, expected result is as follows:
+    # fr => '²&é"(-è_çà)=~1234567890°+',
+    # de => '1234567890ß°!"§$%&/()=?',
+    # cz => ";+ěščřžýáíé=';1234567890%"
+    my $keystrokes = '`1234567890-=!@6~!@#$%^&*()_+';
```

Comment:
I guess that's unwanted change, or?

Reply:
at the moment we don't really use those combinations in the code, we type always in `us` and I thought that the expected result corresponding to selected layout could be just a comment that should match with the needle as well. As far as internally openQA cannot map another language different than `us` we might want to exclude that from code and just leave it as a comment. I found confusing to call the needle `us-keymap` when it should be `cz-keymap` with the previous code. WDYT?

Reply:
Ok, let me clarify that. So how it was before your change: we type symbols which result in the same string in SUT. So single needle can be used to assert all. You apply opposite approach, that we type same thing and assert with multiple needles. Or am I missing something? I personally would go for how it was before only because it's less needles to maintain, but it still has a point that our needles are called in a wrong way. In the end, if it works everywhere, I'm totally fine with the solution, except storing this comment doesn't help much anyways, so I would either remove it or keep a hash map.